content/requests/sign_request_create_request.yml

```diff
@@ -0,0 +1,158 @@
+    description: Disable uploading/adding attachments for signers
+    example: true
+    default: true
+
+  are_text_signatures_enabled:
+    type: boolean
+    description: Disable usage of signatures generated by typing (text)
+    example: true
+    default: true
+
+  is_text_enabled:
+    type: boolean
+    description: Disable ability for signer to add text
+    example: true
+    defaut: true
+
+  are_dates_enabed:
+    type: boolean
+    description: Disable ability for signer to add dates
+    example: true
+    default: true
+
+  are_emails_disabled:
+    type: boolean
+    description: >-
+      Disable all status emails, as well as the original email that
+      contains the sign request
+    example: false
+    defaut: false
+
+  signature_color:
+    type: string
+    enum: ['blue', 'black', 'red']
+    example: blue
+    description: >-
+      Force a specific color for the signature
+
+  is_phone_verification_required_to_view:
+    type: boolean
+    description: >-
+      Forces signers to verify a text message prior to viewing the
+      document. You must specify the phone number of signers to have
+      this setting apply to them.
+    example: true
+
+  email_subject:
+    type: string
+    example: Sign Request from Acme
+    description: Subject of sign request email. This is cleaned by sign
+      request.
+
+  email_message:
+    type: string
+    example: Hello! Please sign the document below
+    description: Message to include in sign request email. This is cleaned,but
+      some html tags are allowed. Links included in the message are
+      also converted to actual links in the email.
+
+  are_reminders_enabled:
+    type: boolean
+    example: true
+    description: >-
+      Automatically remind signers to sign a document on day 3, 8, 13 and
+      18 (or less if the document has been digitally signed already).
+
+  signers:
+    type: array
+    items:
+      $ref: '#/components/schemas/SignRequestCreateSigner'
+    description: Array of signers for the sign request
+
+  source_files:
+    type: array
+    items:
+      $ref: '#/components/schemas/File--Mini'
+    description: List of files to create a signing document from
+
+  parent_folder:
+    allOf:
+        - $ref: "#/components/schemas/Folder--Mini"
+        - description: |-
+            The destination folder to place sign request
+            specific data in (copy of source files, signing log etc.)
+
+  name:
+    type: string
+    example: name
+    description: name of sign request
+
+  prefill_tags:
+    type: array
+    items:
+      $ref: '#/components/schemas/SignRequestPrefillTag'
+    description: When a document contains sign related tags in the content,
+      you can prefill them using this prefill_tags by referencing the 'id'
+      of the tag as the `external_id` field of the prefill tag.
+
+  days_valid:
+    type: integer
+    description: Number of days after which this request will automatically
+      expire if not completed
```

Comment:
If a reminder is after days_valid is it ignored or does the request give an error?

Reply:
I could see people wanting this to be in hours. There are plenty of integers available. If this is a limit of SR at this time we can document only 24 hr increments supported at this time, but then when they do support hours the API doesn't change.

Reply:
This is indeed a limitation at this time. We could change it but maybe not for GA. However, we never got a request this to be less than a day. If there are specific requirements where a doc needs to be signed right away, or not, dev generally use embed_url functionality and direct the user to sign right away. One can also cancel a signature request on demand, without relying on expiring a doc.